### PR TITLE
feat: adding new theme for openapi site and updating docker-compose

### PR DESCRIPTION
**Pull request by brittonhayes**
## What does this do?

- Updates the theme for the open api site
image
- adding docker-compose default ports
- added nodemon setup for local dev with hot reload

### CODE REVIEW

File: src/views/pages/openapidocs.ejs [hunk 1]
```diff
@@ -2,20 +2,74 @@
...
-<body>
-    <rapi-doc spec-url="/swagger/openapi.json" show-header="false" schema-description-expanded="true"
-        primary-color="#83a598" use-path-in-nav-bar="true" show-components="true" default-schema-tab="model"
-        info-description-headings-in-navbar="true" allow-authentication="false" allow-spec-file-download="true">
-        <!-- provide slot attribute to place an element at desired location -->
-        <img slot="nav-logo" src="/public/favicon.ico"
-            style="display:block; margin-left: auto; margin-right: auto; width: 50%; margin-bottom: 20px;" />
+  <body>
+    <rapi-doc
+      spec-url="/swagger/openapi.json"
```
Comment (ecshreve) [inline src/views/pages/openapidocs.ejs:51]: I fought with the formatter in vscode for too long trying to get it to do this 👍

**Review by ecshreve — DISMISSED** [review]
lgtm!

File: nodemon.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "verbose": false,
+    "ignore": [
+        "*.test.*",
+        "dist/*",
+        "node_modules/*"
+    ],
+    "ext": "js,ts,ejs,json,yml"
+}
```
Comment (bagelbits) [inline nodemon.json:9]:
```suggestion
}

```

Comment (bagelbits) [reply nodemon.json:9]: *at myself* You see not \n terminating a file one time cause a rather spectacular bug and you never forget it.

File: src/views/pages/openapidocs.ejs [hunk 3]
```diff
@@ -2,20 +2,74 @@
 
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="utf-8" />
```
Comment (bagelbits) [inline src/views/pages/openapidocs.ejs:6]: You really went hard with the meta tags. Thank you!

**Review by bagelbits — APPROVED** [review]
EXCELLENT!

**Pull request merged**